.github/workflows/python-lint.yaml

```diff
@@ -0,0 +1,32 @@
+name: Linting
+
+on:
+  push:
+    branches: [develop]
```

Comment:
would these not be better as composite workflows using on workflow call

Reply:
Agreed. I'll refine.